.github/workflows/check-multi-package.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+name: Check Multi-package
+
+on:
+  pull_request:
+    branches:
+      - karimeldegwy/*
```

Comment:
Is this a test change? I would think this should apply to all of our core branches, eg `branches: [main, dev, 1.0*, 2.0*, 3.0*, fasttrack/*]`